contracts/extensions/bank/Bank.sol

```diff
@@ -97,11 +97,31 @@ contract BankExtension is AdapterGuard, IExtension {
                     address(this),
                     uint8(flag)
                 ),
-            "bank::accessDenied"
+            string(abi.encodePacked("bank::accessDenied:", toString(flag)))
```

Comment:
Is there a specific reason this is changed here and not in other places? Or is this the only place?